salt/manager/soc_manager.yaml

```diff
@@ -1,4 +1,17 @@
 manager:
+  reposync:
+    enabled:
+      description: This is the daily task of syncing the Security Onion OS packages. It is recommended that you leave this enabled.
+      global: True
+      helpLink: manager.html
+    hour:
+      description: The hour of the day in which the repo sync takes place.
+      global: True
+      helpLink: manager.html 
+    minute:
+      description: The minute within the hour specified in hour to run the repo sync.
```

Comment:
Should wrap "specified in hour" in parens or something. It's difficult to read that.